docs/appActions.md

```diff
@@ -349,6 +349,16 @@ Clears the data specified in dataDetail
 
 
 
+### importBrowserData(selected) 
+
+Import browser data specified in selected
+
+**Parameters**
+
+**selected**: `object`, the browser data to import as per doc/state.md's importBroserDataSelected
```

Comment:
definitely not a big deal, just a misspelling here:
s/importBroserDataSelected/importBrowserDataSelected